.jenkins/pytorch/test.sh

```diff
@@ -11,6 +11,11 @@ source "$(dirname "${BASH_SOURCE[0]}")/common.sh"
 
 echo "Testing pytorch"
 
+export PYTHONIOENCODING=UTF-8
+export LC_ALL=en_US.UTF-8
+export LC_CTYPE=en_US.UTF-8
+export LANG=en_US.UTF-8
```

Comment:
Let's not do it unequivocally for tests. (Let's enable it for one config, but not for all)
Also, please use `locale` command to modify locale rathe that directly setting `LC_` variables

Reply:
Looking to the [manual](https://man7.org/linux/man-pages/man1/locale.1.html) of the command `locale`, I didn't find how to modify the locale. Then I googled about that, and someone mentioned `update-locale`. But I don't know whether it is a built-in program for Linux or not.

Reply:
> Let's not do it unequivocally for tests. (Let's enable it for one config, but not for all)

Are you sure? The test won't take too much time. And it is actually introducing more complexity if we want to do that.

Reply:
Sadly update-locale is not a builtin program and needs to be installed using apt. I’m testing this on ubuntu:18.04 official docker image. The exact step of making python accept UTF-8 content is first set the LC_* ends, then install the package “locales”, and then “update-locale”. I haven’t got a time to take a look at what the exact docker image the CI is using. I feel that only setting the LC_* envs might not be enough.